internal/cli/auth/login.go

```diff
@@ -110,10 +117,67 @@ Your code will expire after %.0f minutes.
 	opts.AuthToken = accessToken.AccessToken
 	opts.RefreshToken = accessToken.RefreshToken
 	opts.SetUpAccess()
+	_, _ = fmt.Fprintf(opts.OutWriter, "Successfully logged in.\n")
+	if opts.noProfile {
+		return opts.config.Save()
+	}
+	if err := opts.InitStore(ctx); err != nil {
+		return err
+	}
+	_, _ = fmt.Fprintf(opts.OutWriter, "Press Enter to continue your profile configuration")
+	_, _ = fmt.Scanln()
+	if err := opts.askOrg(); err != nil {
+		return err
+	}
+	if err := opts.askProject(); err != nil {
+		return err
+	}
+
+	opts.SetUpProject()
+	opts.SetUpOrg()
+	if err := survey.Ask(opts.DefaultQuestions(), opts); err != nil {
+		return err
+	}
+	opts.SetUpOutput()
+	opts.SetUpMongoSHPath()
```

Comment:
could this be a function to make simplify `Run`?

Reply:
having a hard time to come up with a name for this but we could move the oauth part to a function which probably archives the same goal